Libraries/Network/XMLHttpRequestBase.js

```diff
@@ -146,6 +159,11 @@ class XMLHttpRequestBase {
       } else {
         this.responseText += responseText;
       }
+      if(this.reponseType == '' || this.reponseType == 'text')
+          this.response = Object(this.responseText);
```

Comment:
'response' is object type, and Flow was complaining about the string to object type mismatch.